NPC/corner-turret/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "corner-turret"
+version = "0.0.2"
+authors = ["Brooks Townsend<brooksmtownsend@gmail.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+wasmdome-mech-sdk = {path = "../../mech-sdk"}
```

Comment:
For consistency, we should probably use `0.0.7` from the crates website, so our NPCs are using the same version as players on gameday